### DISCUSSION TITLE
Unable to sign up on localhost

### DISCUSSION THREAD

**Discussion by msboffl** [idurar/idurar-erp-crm#904] [General]
@salahlalami 
I'm unable to sign up on localhost, i have setup everything and added all env key - (DATABASE, RESEND_API, OPENAI_API_KEY). Im getting this error while signing up. Is there any local credentials available for localhost..??

<img width="750" alt="Screenshot 2024-01-06 at 11 15 14 AM" src="https://github.com/idurar/idurar-erp-crm/assets/53231064/759a9601-d13b-464a-9037-cd5834a4dcd7">

**Comment by emzm17** [comment]
@msboffl any update on this? are you able to sign in.

**Comment by msboffl** [comment]
No, Im unable to sign up on local

**Reply by emzm17** [reply]
maintainers of this project can help us.

**Comment by msboffl** [comment]
Are you able to sign up on local ??

**Reply by emzm17** [reply]
no

**Comment by msboffl** [comment]
@salahlalami Any update on this .??

**Comment by DeImOs-Sj** [comment]
Same issue have u solved ?

**Comment by msboffl** [comment]
No, I will

**Reply by DeImOs-Sj** [reply]
Once u are done can u ping what was issue in discussion page ?

**Comment by ptminh-kmp** [comment]
@msboffl  , @DeImOs-Sj 
1. Create **Resend** account: [https://resend.com/](https://resend.com/) and create an API KEY.
2. Copy and paste Resend API Key at backend .env: **RESEND_API**
3. Access to MongoDb, schema "settings" and update the following **settingKey**:
- **idurar_registration_allowed**: change **settingValues** from **false** to **true**
- **email_domain**: change **settingValues** to **resend.dev**
- **idurar_app_email**: change **settingValues** to **onboarding@resend.dev**

**Reply by DeImOs-Sj** [reply]
Access to MongoDb, schema "settings" and update the following settingKey:
idurar_registration_allowed: change settingValues from false to true
email_domain: change settingValues to resend.dev
idurar_app_email: change settingValues to [onboarding@resend.dev](mailto:onboarding@resend.dev

i am unable to locate these in the folder structure can u please guide ?

**Reply by ptminh-kmp** [reply]
Just connect to your MongoDB. For me, I use MongoDB Cloud Atlas to access database.
<img width="1141" alt="image" src="https://github.com/idurar/idurar-erp-crm/assets/52888255/fb632c09-25a6-4dcf-bae9-dad867c9356c">
Once you can access database, go to schema **"settings"** and update values.

**Reply by Ritesh0604** [reply]
I am also unable to locate the idurar_registration_allowed setting key.
I have 76 documents in the setting collection, but in your @ptminh-kmp  screenshot, there are 82 documents, why is there a difference?
![Screenshot (75)](https://github.com/idurar/idurar-erp-crm/assets/94890637/5b414641-abaa-4c1c-bbe2-6474efae462c)

**Reply by AhsanSarwar0413** [reply]
@ptminh-kmp I have tried the same but the two things I do have an issue.

1. Its not working even after I go through all of the points. Looks like, the registration doesn't exist anymore in the codebase.
2. There is not such key in any object of settings in DB. `idurar_registration_allowed`

**Comment by DeImOs-Sj** [comment]
![Screenshot from 2024-02-08 11-04-31](https://github.com/idurar/idurar-erp-crm/assets/115064345/a34eaec1-b34a-4b56-be30-bf7c45910942)

I have connected to the database all I am not able to see are the collections and documents so how i amm supposed to update the setting key

**Comment by inesmrad** [comment]
hi,
what i suggest is that you connect the db URL to your localhost (use mongodb compass) , make sure the backend is running then directly login with the provided credentials in the README.md file

**Reply by AliMughal169** [reply]
there are no credentials in README.md. can you share the credentials here?

**Comment by MSNoorul** [comment]
i am unable to see the resigter page 
![image](https://github.com/idurar/idurar-erp-crm/assets/111111285/84dfda4f-1153-4562-9ad4-fe11ba6cb17b)
also not in the source code

**Comment by bruno-keiko** [comment]
I have the same issue, I am unable to log in. Who has accomplished this problem please help me! Thank you!

**Reply by erandakarachchi** [reply]
I think the /register route no longer exists in the codebase. However, its working in the deployed version. 
https://cloud.idurarapp.com/register

Could someone help ?

**Comment by dhuruvandb** [comment]
Hello All,

Simple you can sign in through 

Email - admin@demo.com
Password - admin123

works for me

Before : 

![image](https://github.com/user-attachments/assets/a8feecbb-d44f-4867-876c-74ee3ce7897c)

After : 

![image](https://github.com/user-attachments/assets/f2ec6755-5676-4cbf-85ea-c21b8cbbebcb)

This happened because i just setup the mongodb atlas and it has all these credentials by default when you run : npm run setup

![image](https://github.com/user-attachments/assets/e3ae830e-d9e7-4130-88b7-38a4f8e7f171)

Give it a try.

Happy Coding!

**Reply by avishusa** [reply]
Thanks for sharing this

**Reply by ramGit22** [reply]
Ensure that you have the latest version of node installed!
